### DISCUSSION TITLE
The example code doesn't run

### DISCUSSION THREAD

**Discussion by shi-yan** [Textualize/textual#1357] [Q&A]
run 
```bash
python3 code_browser.py ../
```

as suggested by the doc gives

```bash
Traceback (most recent call last):
  File "/Users/shiy/work/textual/examples/code_browser.py", line 19, in <module>
    from textual.widgets import DirectoryTree, Footer, Header, Static, FileSelected
  File "/Users/shiy/miniconda3/lib/python3.9/site-packages/textual/widgets/__init__.py", line 55, in __getattr__
    raise ImportError(f"Package 'textual.widgets' has no class '{widget_class}'")
ImportError: Package 'textual.widgets' has no class 'FileSelected'
```

**Comment by davep** [comment]
Presumably you have a copy of `Textual` installed, perhaps via `pip`? And you then pulled the code down from GitHub to test out the examples? If that is the case can you confirm what version of Textual you have installed, and when you cloned the code to get the examples?

Also, what happens if you try `python3 -m textual`? If the demo runs up, what is the version number towards the bottom of the sidebar (press Ctrl+b to make it appear).

Any other information about how you set up Textual and the examples would be very helpful in helping you.

Note that, testing here, with 0.6.0, I'm running the examples just fine.
